prow/cluster/deck_deployment.yaml

```diff
@@ -46,6 +46,7 @@ spec:
         - --redirect-http-to=prow.k8s.io
         - --oauth-url=/github-login
         - --job-config-path=/etc/job-config
+        - --spyglass=true
```

Comment:
Are we ready for this?

Reply:
There are no front end Deck UI components added yet, thus most users will still go straight to gubernator for viewing jobs. This is only to enable spyglass links for people who would like to use them. Deck UI components for fully adopting spyglass will be added once the viewers are production-ready.

Reply:
I think we'll need to add the flag after the image bump, or else the deployment will be invalid. Or we can bump right after this is merged